src/Views/sedimentview.html

```diff
@@ -12,15 +12,14 @@ <h4 class="modal-title">Sediment Machine Learning Method</h4>
     <div class="modal-body">
         <div class="panel">
             <!-- https://github.com/fragaria/angular-daterangepicker -->
-            <!-- <small>Example to be changed for sediment model</small>
-            <h4 class="panel-title">Select/Enter a Date Range<small style="color:red">&nbsp;&nbsp;(Required)</small></h4>
-            <input id="dateSimple" date-range-picker class="form-control date-picker" type="text" ng-model="vm.dateRange.dates" min="vm.dateRange.minDate" max="vm.dateRange.maxDate" options="vm.dateRangeOptions" /> -->
+            <h4 class="panel-title">Select/Enter a Date Range<small style="color:red">&nbsp;&nbsp;(Required) &nbsp;&nbsp;</small><span><i title="Date range is limited to 2006 and later due to model limitations." class="fa fa-question-circle"></i></span></h4>
```

Comment:
For consistency across the client, can we use this tooltip method?

Example: https://github.com/USGS-WiM/StreamStats/blob/dev/src/Views/batchprocessor.html#L9L12

You can view in the client by clicking the Batch Processor button at top right and hovering over the icon on the header:

![image](https://github.com/USGS-WiM/StreamStats/assets/40237491/9994837a-1f9e-43e2-970b-ee9355e37c58)